.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+name: ansible-lint
+on:
+  pull_request:
+    branches: ["main", "stable-*"]
+jobs:
+  build:
+    name: Ansible Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run ansible-lint
+        uses: ansible/ansible-lint-action@v6
```

Comment:
You probably need to check out the repository first, so the linter has something to run on :)

Reply:
I would have guessed so too, the official documentation says otherwise: https://ansible-lint.readthedocs.io/installing/#installing-ansible-lint-as-a-github-action

Reply:
The official documentation seems to be pretty out of date.